charts/testkube/values.yaml

```diff
@@ -114,6 +116,9 @@ mongodb:
 ## Tolerations to schedule a workload to nodes with any architecture type. Required for deployment to GKE cluster.
 ## More info: https://cloud.google.com/kubernetes-engine/docs/how-to/prepare-arm-workloads-for-deployment#node-affinity-multi-arch-arm
 nats:
+  natsbox:
```

Comment:
this should be first tested and specified in our `values-develop.yaml`, `values-stage.yaml`, `values-demo.yaml` files as this can introduce a breaking change for our users that already have Testkube installed. Therefore, it is better to try deploying it on our clusters first.

Reply:
moved it just for demo